src/code.less

```diff
@@ -25,4 +25,15 @@ pre {
     border-radius: 3px;
     overflow-x: auto;
     white-space: pre-wrap;
-}
\ No newline at end of file
+}
+pre code{
+    color: inherit;
+    background: transparent;
+    margin-top: -1.50em;
```

Comment:
Why is this `margin-top` here? This will surely break sites using PaperCSS.

Reply:
This is because there is somehow an extra line left initially when we use pre code. So the solution to this is adding a negative margin equal to the line height. Hence, the `margin-top`
refer: https://stackoverflow.com/a/16240981/5536613

Reply:
This is very far from standard. What happens when someone does this?

```
<pre><code>console.log('your code goes here')</code></pre>
```

Then the that negative margin-top will pull that out of the `pre` area. And exactly this will happen when someone will use markdown parsers for parsing codeblocks wrapped with _triple backtics_.

The problem is that, the current `<pre><code>` format  in the `index.html` file is not ideal. Codes should start right after the starting `code` tag, and not from the next line.

Reply:
You can check GitHub's `<pre><code>` css style by inspecting it from my previous comment. Add that negative margin-top there from your browser's developer tools. You'll see what I meant.

Reply:
Thank you! I think I've understood your point and have made the necessary changes. What do you think now?